Crypto/PublicKey.cc

```diff
@@ -30,7 +30,7 @@ namespace litecore { namespace crypto {
     using namespace fleece;
     using namespace net;
 
-    LogDomain TLSLogDomain("TLS", LogLevel::Warning);
+    LogDomain TLSLogDomain("TLS");
```

Comment:
How much noise at the INFO level? Do we need to enable this at the platform level?

Reply:
What I have noticed, without the initial log level, it will initialize with what level platform has defined. 

When initialized with `LogLevel::Warning` level, its level  always returns `LogLevel::Warning` [here](https://github.com/couchbase/couchbase-lite-core/blob/master/LiteCore/Support/Logging.cc#L406)